src/vulkan-renderer/renderer.cpp

```diff
@@ -187,38 +173,40 @@ VkResult VulkanRenderer::record_command_buffers() {
     for (std::size_t i = 0; i < swapchain->get_image_count(); i++) {
         spdlog::debug("Recording command buffer #{}.", i);
 
+        auto current_command_buffer = command_buffers[i]->get();
```

Comment:
It isn't too clear what command buffer is here, i think `VkCommandBuffer *` should be used instead.

Reply:
Done.